.husky/pre-commit

```diff
@@ -3,4 +3,8 @@
 
 npm run lint:fix
 
+npx pretty-quick --staged
+
+npx tsc --noEmit # Ensure that code compiles
```

Comment:
One issue I had during testing was that formatting and linting changes would reposition code comments, sometimes resulting in `// @ts-ignore` and `// eslint-disable-foo` being invalidated. Thus, checking that code compiles on precommit can be a good idea.